src/components/IstioWizards/Slider/Slider.tsx

```diff
@@ -165,15 +174,7 @@ class Slider extends React.Component<Props, State> {
             <PlusIcon />
           </Button>
           <InputGroupText>{this.props.inputFormat}</InputGroupText>
-          {this.props.showLock && (
-            <Button
-              className={pinButtonStyle}
-              variant={this.props.locked ? ButtonVariant.primary : ButtonVariant.secondary}
-              onClick={() => this.props.onLock(!this.props.locked)}
-            >
-              <ThumbTackIcon />
-            </Button>
-          )}
+          {this.props.showLock ? LockIcon : <></>}
```

Comment:
Probably, use `null` instead of `<></>`?

Reply:
```ts
{this.props.showLock && LockIcon}
```

Reply:
That does not work because showLock is not `null`, just `false`, unfortunately.

Reply:
If showLock is a Boolean you can do it, in fact if showLock is false or null the component won't render  LockIcon

Reply:
```
TypeScript error in /home/cfcosta/Projects/go/src/github.com/kiali/kiali-ui/src/components/IstioWizards/Slider/Slider.tsx(177,11):
Type 'false | Element' is not assignable to type 'Element'.
  Type 'false' is not assignable to type 'Element'.  TS2322
```

Nope, it does not work :(

Reply:
Mm but the prop is Boolean not element.

```ts
type Props = {
  id: string;
  orientation: string;
  min: number;
  max: number;
  maxLimit: number;
  step: number;
  value: number;
  tooltip: boolean;
  onSlide: (value: number) => void;
  input: boolean;
  sliderClass: string;
  inputFormat: string;
  locked: boolean;
  showLock: boolean;
  onLock: (locked: boolean) => void;
};
```

Reply:
Yes, but the `LockIcon` is.

Reply:
Do we still have an issue here? @aljesusg  I think we are waiting on your approval.

Reply:
It's not a blocker @mtho11 , It's an improvement, feel free to merge this.

My only comment is that LockIcon is always true because is a Element defined, so {this.props.showLock && LockIcon} should work

Reply:
I'm going to merge this then and we can change this later if necessary.